src/components/layout/Sidebar/index.jsx

```diff
@@ -54,6 +55,7 @@ export default class Sidebar extends React.Component {
         </button>
 
         <div className="sidebar-content">
+          <Breadcrumbs items={this.props.breadcrumbs}/>
```

Comment:
It shouldn't be here, it should be a child of the sidebar. Imagine that you want to re-use the sidebar, do you want to re-instantiate the breadcrumbs?